### ISSUE TITLE
Failure to create default path to database url

### ISSUE THREAD

**Issue by malomehi** [Flexget/Flexget#2759]
### Expected behaviour:

The default path to the database uri is generated correctly and is not an empty string.

### Steps to reproduce:
Start the deamon, for example with `flexget --loglevel verbose daemon start`

  
#### Log:
<details>
 <summary>(click to expand)</summary>

```
Traceback (most recent call last):
  File "/usr/local/bin/flexget", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/flexget/__init__.py", line 44, in main
    manager.start()
  File "/usr/local/lib/python3.8/dist-packages/flexget/manager.py", line 372, in start
    self.initialize()
  File "/usr/local/lib/python3.8/dist-packages/flexget/manager.py", line 224, in initialize
    self.init_sqlalchemy()
  File "/usr/local/lib/python3.8/dist-packages/flexget/manager.py", line 807, in init_sqlalchemy
    self.engine = sqlalchemy.create_engine(
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/__init__.py", line 479, in create_engine
    return strategy.create(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/strategies.py", line 54, in create
    u = url.make_url(name_or_url)
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/url.py", line 229, in make_url
    return _parse_rfc1738_args(name_or_url)
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/url.py", line 290, in _parse_rfc1738_args
    raise exc.ArgumentError(
sqlalchemy.exc.ArgumentError: Could not parse rfc1738 URL from string ''
```
</details>

### Additional information:

- FlexGet version: 3.1.78
- Python version: 3.8.6
- Installation method: pip
- Using daemon (yes/no): yes
- OS and version: Ubuntu 16.04.7 LTS
- Link to crash log:

Potential fix proposal:

The issue is here:
https://github.com/Flexget/Flexget/blob/develop/flexget/manager.py#L795

This should compare to an empty string and not to `None` replacing `if self.database_uri is None:` with `if self.database_uri == '':`

**Comment by BrutuZ** [comment]
To be fixed by #2758

**Comment by gazpachoking** [comment]
Fix was merged.
